localstack/services/stepfunctions/asl/parse/test_state/preprocessor.py

```diff
@@ -48,20 +48,20 @@ def _decorated_updates_inspection_data(method, inspection_data_key: InspectionDa
     def wrapper(env: TestStateEnvironment, *args, **kwargs):
         method(env, *args, **kwargs)
         result = to_json_str(env.stack[-1])
-        env.inspection_data[inspection_data_key.value] = result  # noqa
+        env.inspection_data[inspection_data_key.value] = result  # noqa: we know that the here value is a supported inspection data field by design.
 
     return wrapper
 
 
 def _decorate_state_field(state_field: CommonStateField) -> None:
     if isinstance(state_field, ExecutionState):
         state_field._eval_execution = _decorated_updates_inspection_data(
-            method=state_field._eval_execution,
-            inspection_data_key=InspectionDataKey.RESULT,  # noqa
+            method=state_field._eval_execution,  # noqa: as part of the decoration we access this protexted member.
```

Comment:
super nit: protected

FYI: you can disable individual [rules](https://docs.astral.sh/ruff/rules/) using `# noqa: F401` (example for unused imports)